content/blog/pragmatic-architecture.md

```diff
@@ -0,0 +1,61 @@
+
+## What makes an Architect?
+Let's assume that not all Architects are pretentious douchebags, but let's instead look at what makes an Architect. I consider everyone who spends time on architecture an Architect. Whether you're working on software, system design, infrastructure, or data doesn't matter. You're involved in the process of defining and evolving architecture. So what do you need to do that?
+
+Of course you need a solid technical background to start from, ideally with a good range of different technologies. You also need to be able to grasp things quickly. You need to understand why things work, and the pros and cons of different things. But just being technically and analytically strong doesn't get you there.
+
+You need to be able to work towards a good solution, but half the information is missing, incomplete, or downright untrue. And it needs to be done next week. There's no time or budget to _properly_ assess all options, or prototype candidate solutions. And some of the stakeholders will not be available for a chat until next month. Great.
+
+A good Architect knows which questions to ask, which 'information' to question or disregard, how to 'read between the lines', and how to make educated guesses. A huge part of that is building experience, reviewing other architects' work, and becoming good at pattern recognition. Recognizing patterns allows an architect to use their experience to apply certain architectural patterns or guidelines to come up with a (most likely) fitting solution quickly without having to reinvent the wheel each time.
+
+Ultimately though, Architects are just people, and as such they have human flaws; tunnel vision being one of them. When you've applied a certain type of solution successfully a number of times, you're more likely to choose it again. Even in cases where it might not be the best solution. Add in a pinch of pride, and you're well underway to become a Dogmatic Architect.
+
+## When does an Architect become a Dogmatic Architect?
+The Dogmatic Architect will typically be over-confident and stubborn. They will propose a solution, based on either a particular Architectural Principle or based on (anecdotal) historical evidence. But don't be too quick to label someone as dogmatic. None of the signs I just described can be measured empirically. So it's just _your_ perspective of that person. They might just feel you don't need to get the full low-down of 3 months of gathering requirements and constraints, and have compressed their story more than you wanted. They might be prioritizing different principles of guidelines than you would normally do. And if that makes someone dogmatic, what does it make you?
+
+I'm not a big fan of labelling people, and especially with these kinds of labels I prefer to tread carefully. An Architect coming across as 'somewhat dogmatic' can be a matter of perspective, different communication styles, or it can be a sign that they are, in fact, a bit stuck in their ways.
```

Comment:
De tweede zin loopt wat ongemakkelijk. Suggestie:

```
An Architect coming across as 'somewhat dogmatic` can simply be a matter of perspective, different communication styles, a differing personality. It could also be a sign that they are simply stuck in their ways and would do well to broaden their perspective.
```